pkg/agent/multicast/mcast_controller.go

```diff
@@ -57,17 +58,19 @@ var (
 )
 
 type mcastGroupEvent struct {
-	group net.IP
-	eType eventType
-	time  time.Time
-	iface *interfacestore.InterfaceConfig
+	group  net.IP
+	eType  eventType
+	time   time.Time
+	iface  *interfacestore.InterfaceConfig
+	sender net.IP
```

Comment:
does sender mean a sender in multicast group? what if there are multiple senders?

Reply:
No it means the source Node who sends IGMP report. I rename it as "srcNode". It is used only with encap mode.